### PR TITLE
Feature/retrieve record updates and examples

### PR DESCRIPTION
**Pull request by declankieran-nhsd**
@vickyjaiswal0308 @marcusfearnett-nhsdigital For review

This is just cut straight off the branch from the last PR, so the diff between that branch and this gives a proper set of changes.

Changed the retrieve record use case page to match the add record, and consent use case pages in terms of diagrams styles and page structure.

### CODE REVIEW

File: input/fsh/codesystem/CPIS-ConditionCategory.fsh
Comment (vickyjaiswal0308) [inline input/fsh/codesystem/CPIS-ConditionCategory.fsh:1]: I think the name should be changed from "ReasonableAdjustmentsConditionCategory" to "CPISConditionCategory"

File: input/fsh/codesystem/FGM-ConditionCategory.fsh
Comment (vickyjaiswal0308) [inline input/fsh/codesystem/FGM-ConditionCategory.fsh:1]: I think the name should be changed from "ReasonableAdjustmentsConditionCategory" to "FGMConditionCategory"

File: input/fsh/codesystem/FGM-FlagCategory.fsh
Comment (vickyjaiswal0308) [inline input/fsh/codesystem/FGM-FlagCategory.fsh:1]: name should be changed to "FGMFlagCategory"

File: input/fsh/profiles/FlagCondition.fsh
Comment (vickyjaiswal0308) [inline input/fsh/profiles/FlagCondition.fsh:1]: name should be changed to generic "FlagCondition"

Comment (declankieran-nhsd) [reply input/fsh/profiles/FlagCondition.fsh:1]: Done

File: input/fsh/codesystem/FGM-ConditionCategory.fsh
Comment (declankieran-nhsd) [reply input/fsh/codesystem/FGM-ConditionCategory.fsh:1]: Cheers Vicky, done

File: input/fsh/codesystem/FGM-FlagCategory.fsh
Comment (declankieran-nhsd) [reply input/fsh/codesystem/FGM-FlagCategory.fsh:1]: Done

File: input/fsh/codesystem/CPIS-ConditionCategory.fsh
Comment (declankieran-nhsd) [reply input/fsh/codesystem/CPIS-ConditionCategory.fsh:1]: Done

File: input/fsh/valuesets/FlagConditionCode.fsh
Comment (vickyjaiswal0308) [inline input/fsh/valuesets/FlagConditionCode.fsh:1]: name should be changed to FlagConditionCode

File: input/fsh/valuesets/FlagConditionCategory.fsh
Comment (vickyjaiswal0308) [inline input/fsh/valuesets/FlagConditionCategory.fsh:1]: name should be changed to FlagConditionCategory

File: input/fsh/valuesets/RA-ProvenanceRole.fsh
Comment (vickyjaiswal0308) [inline input/fsh/valuesets/RA-ProvenanceRole.fsh:1]: It seems confusing whether we should have generic FlagProvenanceRole or RAProvenanceRole as the title and name doesen't match with the ValueSet and Url.

Comment (declankieran-nhsd) [reply input/fsh/valuesets/RA-ProvenanceRole.fsh:1]: Yes, sorry missed that as well.  I had already updated the canonical to be FlagProvenanceRole, but hadn't change the file name etc.  Cheers, I've done this now.    This very much could just be ProvenanceRole in an NHSE context, same with the profiles, but I'll keep the Flag prefix for now until we talk to the NHSE team to see how these might be adopted

File: input/fsh/valuesets/FlagConditionCategory.fsh
Comment (declankieran-nhsd) [reply input/fsh/valuesets/FlagConditionCategory.fsh:1]: Done

File: input/fsh/valuesets/FlagConditionCode.fsh
Comment (declankieran-nhsd) [reply input/fsh/valuesets/FlagConditionCode.fsh:1]: Done

**Pull request merged**